platform/darwin/src/MGLStyleAttributeValue_Private.hpp

```diff
@@ -17,4 +19,7 @@
 - (mbgl::style::PropertyValue<std::vector<float> >)mbgl_numberArrayPropertyValue;
 - (mbgl::style::PropertyValue<uint8_t>)mbgl_enumPropertyValue;
 
+// Convert mbgl types to darwin types
+- (id <MGLStyleAttributeValue>)mbgl_colorPropertyValueWith:(mbgl::style::PropertyValue<mbgl::Color>)color;
```

Comment:
I think we're both off track 😄 Since objc doesn't support generics, naming a method with a generic type param is very confusing. How about `-mbgl_colorWithPropertyValue:` which theoretically should be `-mbgl_colorWithPropertyValueColor:` but that seems excessive. 
same goes for all methods in [MGLStyleAttribute.hpp](https://github.com/mapbox/mapbox-gl-native/blob/5626-runtime-styling/platform/darwin/src/MGLStyleAttribute.hpp)